### ISSUE TITLE
Question/correction regarding example https://co-cddo.github.io/ukgov-metadata-exchange-model/assets/examples/Dataset-os-postcodes-datset.yaml.json

### ISSUE THREAD

**Issue by PeterParslow** [co-cddo/ukgov-metadata-exchange-model#69]
Feedback on [Dataset](https://co-cddo.github.io/ukgov-metadata-exchange-model/Dataset)

**Describe the bug**
It's great to give examples, and I'm fine with them being in "RDF / YAML / JSON" format.

My question concerns how the examples were created and whether this on in particular could be fixed. It declares its similarity to https://www.data.gov.uk/dataset/2dfb82b4-741a-4b93-807e-11abb4bb0875/os-postcodes-data and that appears to be the source of much of it.

BUT: the MEM example states that the OS CodePoint Open dataset is published by "department-for-work-pensions" whereas the Data.gov.uk source states (correctly) that it is published by Ordnance Survey. 

**To Reproduce**
1. Go to 

- either MEM / Model and scroll down to examples, or
- directly to https://[co-cddo.github.io/ukgov-metadata-exchange-model/assets/examples/Dataset-os-postcodes-datset.yaml.json](https://co-cddo.github.io/ukgov-metadata-exchange-model/assets/examples/Dataset-os-postcodes-datset.yaml.json)

3. look at the "publisher" property
4. compare it to the publisher shown if you browse to the URL given as "identifier"
5. See error

**Expected behavior**
Credit dataset to correct publisher

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by AlasdairGray** [comment]
You are right, it would be clearer to a wider set of users if we updated this example to be published by OS.

The background here is that this was originally based on a record in the DWP catalogue, but was then supplemented with information from the data.gov.uk entry in order to provide realistic values for each of the properties in the metadata exchange model. It would be cleaner and more accurate to update the rest of the values to be based on the data.gov.uk entry rather than some weird hybrid record.
